src/coreclr/jit/emitarm64.cpp

```diff
@@ -8184,6 +8257,68 @@ void emitter::emitIns_R_R_R(
             fmt = IF_SVE_AE_3A;
             break;
 
+        case INS_sve_andv:
+        case INS_sve_eorv:
+        case INS_sve_orv:
+            assert(isFloatReg(reg1));
+            assert(isLowPredicateRegister(reg2));
+            assert(isVectorRegister(reg3));
+            assert(insOptsScalableToSimdScalar(opt));
+            fmt = IF_SVE_AF_3A;
+            break;
+
+        case INS_sve_andqv:
+        case INS_sve_eorqv:
+        case INS_sve_orqv:
+            unreached(); // TODO-SVE: Not yet supported.
```

Comment:
likewise here.

Reply:
can any of these instructions block us from the basic scenario that we are thinking of enabling?